cmd/cloud/deploy.go

```diff
@@ -108,5 +109,5 @@ func deploy(cmd *cobra.Command, args []string) error {
 	// Silence Usage as we have now validated command input
 	cmd.SilenceUsage = true
 
-	return deployImage(config.WorkingPath, deploymentID, ws, pytestFile, envFile, imageName, deploymentName, forcePrompt, dags, astroClient)
+	return deployImage(config.WorkingPath, deploymentID, ws, pytestFile, envFile, imageName, deploymentName, dagDeploy, forcePrompt, dags, astroClient)
```

Comment:
nitpick: should we validate the value of `dagDeploy` before passing it to the deployImage, just curious on what would happen in case someone passes an invalid value.

Reply:
Right now, it would maintain the current state and not deploy it. But i think it makes sense to validate the incoming value and throw an error if the value is not one among the possible values.